locales/en-US/translation.yml

```diff
@@ -229,67 +229,88 @@ search:
     results: '{{value}} results'
 
 admin:
+  top:
+    title: Wiki Admin
+    description: This page can only be accessed by the Wiki administrators.
+    appoint_admin: You can appoint a user as a Wiki administrator using the Make Admin button in Manage User page.
+  navigation:
+    top: Admin Info
+    title: Admin Page
+    app: App Settings
+    notification: Notification
```

Comment:
Suggested change (if this is where all notifications will be, not just 1): Notifications